controllers/adminC.js

```diff
@@ -0,0 +1,26 @@
+const Admin = require('../models/admin');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const asyncWrapper = require('../middleware/async');
+const { createCustomError } = require('../errors/custom_error');
+
+const createAdmin = asyncWrapper(async (req, res, next) => {
+    const { firstname, lastname, email, password } = req.body
+    const adminExist = await Admin.findOne({email});
+    if (adminExist) {
+        return next(createCustomError("Admin already exist", 403));
+    }
+    const salt = bcrypt.genSaltSync(10);
+    const hashPassword = bcrypt.hashSync(password, salt);
+
+    const newAdmin = await Admin.create({
+        firstname,
+        lastname,
+        email,
+        password: hashPassword
+    })
+
+    res.status(201).json({ msg: 'Admin created successfully', admin: newAdmin });
+});
+
+module.exports = { createAdmin }
```

Comment:
You can see red line at the end of all the file your created.
It is because you didn't add and extra line after your implementation.
Make it a habit to leave an extra line always.
```suggestion
module.exports = { createAdmin }

```